src/main/kotlin/com.bugsnag.android.gradle/AndroidManifestInfo.kt

```diff
@@ -9,11 +9,11 @@ import java.io.File
 
 @JsonClass(generateAdapter = true)
 data class AndroidManifestInfo(
-    var apiKey: String,
-    var versionCode: String,
-    var buildUUID: String?,
-    var versionName: String,
-    var applicationId: String
+    val apiKey: String,
+    val versionCode: String,
+    val buildUUID: String?,
+    val versionName: String,
+    val applicationId: String
```

Comment:
I assumed these were unintentionally `var`

Reply:
Yep - I don't think there's any reason for these to not be `val`.